### PR TITLE
fix: make small tweaks to home-page

### PR DESCRIPTION
**Pull request by shimks**
Fixes:
- background image not being expanded to fit the entire screen
- cards not being clickable
- card not having the correct colour
- button in the footer not being `btn-primary`
- `btn-secondary` not having the correct font style
- links in the navbar not turning to the correct colour when hovered over
- change font weight of `deverloper preview 2` logo to semibold
- description and image in features not having enough space between them
- header and description in features padding

### CODE REVIEW

File: css/loopback.css [hunk 1]
```diff
@@ -1,275 +1,309 @@
...
+  border-bottom-width: 0px;
+  border-left-width: 0px;
+  border-right-width: 0px;
+  box-shadow: 0px 4px 6px 0px rgba(0, 0, 0, 0.15);
+  text-align: left;
+  vertical-align: middle;
+  flex: 1;
+  color: var(--color-gray-xd);
+  cursor: pointer;
+}
```
Comment (virkt25) [inline css/loopback.css]: New line after.

File: css/loopback.css [hunk 2]
```diff
@@ -1,275 +1,309 @@
...
+  font-weight: bold;
+  line-height: 16px;
 }
 
 .btn-primary:active,
 .btn-primary:focus,
 .btn-primary:hover {
-    background-color: var(--color-white) !important;
-    color: var(--color-blue-xd);
+  background-color: var(--color-white) !important;
```
Comment (virkt25) [inline css/loopback.css]: Is the `!important` really needed?

File: css/loopback.css [hunk 3]
```diff
@@ -1,275 +1,309 @@
...
+  color: var(--color-blue-xd);
 }
 
 .btn-secondary:active,
 .btn-secondary:focus,
 .btn-secondary:hover {
-    background-color: var(--color-blue-d) !important;
-    color: var(--color-white);
+  background-color: var(--color-blue-d) !important;
```
Comment (virkt25) [inline css/loopback.css]: Same thing. Is the `!important` really needed?

File: css/loopback.css (hunk 1 above)
Comment (shimks) [reply css/loopback.css]: done

**Pull request merged**